src/Sylius/Bundle/CoreBundle/Commander/UpdateVariantsCommander.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Commander;
+
+use Sylius\Component\Product\Command\UpdateVariants;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class UpdateVariantsCommander implements UpdateVariantsCommanderInterface
```

Comment:
`Commander` is semantically correct but seems... weird 😄 What about `UpdateVariantsComandDispatcher`? What about adding `Batched` somewhere to indicate we're operating on badges here?

Reply:
But lets add `Batched` only to the class implementation, as this is just detail
```suggestion
final class BatchedUpdateVariantsCommander implements UpdateVariantsCommanderInterface
```